src/products/controller/products.controller.ts

```diff
@@ -35,6 +43,7 @@ export class ProductsController {
     );
   }
 
+  @UseGuards(JwtGuard, RolesGuard)
```

Comment:
Here you could also add `@Roles(Role.ADMIN, Role.CUSTOMER)`, even though it also works without it, just to explicitly specify the allowed roles.